metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -98,8 +104,11 @@ class KafkaConnectSourceConfig(
         default=[],
         description="Provide lineage graph for sources connectors other than Confluent JDBC Source Connector, Debezium Source Connector, and Mongo Source Connector",
     )
-
     stateful_ingestion: Optional[StatefulStaleMetadataRemovalConfig] = None
+    include_column_lineage: bool = Field(
+        default=False,
+        description="Populates topic->table and table->topic column lineage.",
```

Comment:
```suggestion
        description="Populates column lineage for source table->topic and topic->destination table edges. Works only for SnowflakeSinkConnector.",
```

How easy it is to support this for all connectors ?